packages/core/database/lib/metadata/relations.js

```diff
@@ -433,6 +444,11 @@ const createJoinTable = (metadata, { attributeName, attribute, meta }) => {
         name: `${joinTableName}_inv_fk`,
         columns: [inverseJoinColumnName],
       },
+      {
+        name: `${joinTableName}_unique`,
+        columns: [joinColumnName, inverseJoinColumnName],
+        type: 'unique',
+      },
     ],
```

Comment:
This is more of a question regarding metadata.
What I understood about it was that it is an object that represents the schema in the database + other info.

Are you creating an index by adding this?

Reply:
yes exactly :) it's what is used to create the database tables